smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/protocols/cbor/SmithyRpcV2Cbor.java

```diff
@@ -0,0 +1,334 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.typescript.codegen.protocols.cbor;
+
+import java.util.Set;
+import java.util.TreeSet;
+import software.amazon.smithy.aws.traits.protocols.AwsQueryCompatibleTrait;
```

Comment:
Can't remember if there was some discussion on why we need this here. Is there any way around it?

Reply:
There's some conditional generated code if the trait is present, and the spec says to handle this trait, so I don't see how we avoid this line?

Reply:
note: remove AwsQueryCompat from here, implement in extending class in JSv3